### ISSUE TITLE
ytdl.getInfo(...).videoDetails.title does not always honor lang

### ISSUE THREAD

**Issue by MarmadileManteater** [fent/node-ytdl-core#1136]
`ytdl.getInfo` does not always return the correct localized title even when it is provided the `lang` option. [I created a gist](https://gist.github.com/MarmadileManteater/5e59345b205e1480ace104d3ee9e0c50) to demonstrate this using the localized response from `ytpl` as a reference and using the videos in this playlist: [PLB6Mfu4v-kdUKpNVGykSxx5CpnTP2_rcB](https://youtube.com/playlist?list=PLB6Mfu4v-kdUKpNVGykSxx5CpnTP2_rcB)

![localization](https://user-images.githubusercontent.com/106682128/188250240-c233921c-b46e-4168-b000-0b9da6094a7a.gif)
*(in this example `playlistResponse` comes from `ytpl`)*

As I dug into the issue further, I discovered that the correct localized title is stored in the response object, but it is buried.
This is where the expected values of `.videoDetails.title` and `.videoDetails.description` are located.

`.videoDetails.title` should be the value of `.response.contents.twoColumnWatchNextResults.results.results.contents[0].videoPrimaryInfoRenderer.title.runs[0].text`

![image](https://user-images.githubusercontent.com/106682128/188250296-12f98aac-83e9-4fb5-8cf4-19df0b2a605a.png)

`.videoDetails.description` should (probably) be the value of `.response.contents.twoColumnWatchNextResults.results.results.contents[1].videoSecondaryInfoRenderer.description.runs[0].text`
*(update `.response.contents.twoColumnWatchNextResults.results.results.contents[1].videoSecondaryInfoRenderer.description.runs` contains an array of the lines in the localized description)*

[I created another gist](https://gist.github.com/MarmadileManteater/37b48376fcf250852f704fa1766bd4fa) to demonstrate using this workaround for the `title` field.

TLDR; the correct localized title and description are being fetched by `ytdl.getInfo`, but `ytdl.getInfo(...).videoDetails` doesn't contain them, and instead, they are buried in the response object.
